cloudmanager/automation_config.go

```diff
@@ -105,17 +105,17 @@ type SSL struct {
 
 // Auth authentication config
 type Auth struct {
-	AuthoritativeSet         bool                     `json:"authoritativeSet"`
 	AutoAuthMechanism        string                   `json:"autoAuthMechanism"`
 	AutoUser                 string                   `json:"autoUser,omitempty"`
 	AutoPwd                  string                   `json:"autoPwd,omitempty"`
 	DeploymentAuthMechanisms []string                 `json:"deploymentAuthMechanisms"`
-	Disabled                 bool                     `json:"disabled"`
 	Key                      string                   `json:"key,omitempty"`
 	Keyfile                  string                   `json:"keyfile,omitempty"`
 	KeyfileWindows           string                   `json:"keyfileWindows,omitempty"`
 	UsersDeleted             []interface{}            `json:"usersDeleted"`
 	UsersWanted              []map[string]interface{} `json:"usersWanted"`
+	AuthoritativeSet         bool                     `json:"authoritativeSet"`
+	Disabled                 bool                     `json:"disabled"`
```

Comment:
Were these meant to be re-arranged? It seems they were previously in alphabetical order

Reply:
Yes, due to how memory allocation works on go these saves a couple of bytes, https://medium.com/@sebassegros/golang-dealing-with-maligned-structs-9b77bacf4b97